include/rocksdb/statistics.h

```diff
@@ -470,11 +470,18 @@ class Statistics {
     return type < HISTOGRAM_ENUM_MAX;
   }
 
+  virtual double getHistogramPercentile(uint32_t type,
+                                        double percentile) const = 0;
+  virtual uint64_t getHistogramSum(uint32_t type) const = 0;
```

Comment:
If we expose a getHistogram() method, we might have to support methods modifying histogram data like Clear(), Empty() and Add() as well, for example like getHistogram(tickerType)->Add(10). This would not work very well with the core-local statistics infrastructure.